app/src/main/java/com/github/h3lp3rs/h3lp/HelpParametersActivity.kt

```diff
@@ -23,6 +30,10 @@ class HelpParametersActivity : AppCompatActivity() {
     // userLocation contains the user's current coordinates (is initialized to null since we could
     // encounter an error while getting the user's location)
     private var userLocation: Location? = null
+    private var latitude: Double? = null
+    private var longitude: Double? = null
```

Comment:
Instead of initializing these variables to null, you could use the lateinit modifier